src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -325,14 +325,35 @@ std::string ContextAttribute::renderAsJsonObject
   {
     if (omitValue == false)
     {
+      std::string effectiveValue = "";
+      bool valueIsNumberOrBool = false;
+      switch(valueType)
```

Comment:
style: align variables (and '=') and leave an empty line between vars and code.
Also, spece between 'switch' and '('   (switch isn't a function)